DESCRIPTION

```diff
@@ -1,90 +1,69 @@
+           comment = c(ORCID = "0000-0003-1878-3253")),
+    person("Kevin", "Kunzmann", , "kevin.kunzmann@boehringer-ingelheim.com", role = "aut",
+           comment = c(ORCID = "0000-0002-1140-7143")),
+    person("Yoni", "Sidi", , "yoni.sidi@sagerx.com", role = "aut"),
+    person("Christian", "Stock", , "christian.stock@boehringer-ingelheim.com", role = "aut",
+           comment = c(ORCID = "0000-0002-3493-3234")),
+    person(, "Eli Lilly and Company", role = c("cph", "fnd")),
+    person("Boehringer Ingelheim Pharma GmbH & Co. KG", role = c("cph", "fnd"))
+  )
+Description: The mixed model for repeated measures (MMRM) is a popular
+    model for longitudinal clinical trial data with continuous endpoints,
+    and 'brms' is a powerful and versatile package for fitting Bayesian
+    regression models.  The 'brms.mmrm' R package leverages 'brms' to run
+    MMRMs, and it supports a simplified interfaced to reduce difficulty
+    and align with the best practices of the life sciences.  References:
+    Bürkner (2017) <doi:10.18637/jss.v080.i01>, Mallinckrodt (2008)
+    <doi:10.1177/009286150804200402>.
 License: MIT + file LICENSE
-URL:
-  https://openpharma.github.io/brms.mmrm/,
-  https://github.com/openpharma/brms.mmrm
+URL: https://openpharma.github.io/brms.mmrm/,
+    https://github.com/openpharma/brms.mmrm
 BugReports: https://github.com/openpharma/brms.mmrm/issues
-Authors@R: c(
-  person(
-    given = c("William", "Michael"),
-    family = "Landau",
-    role = c("aut", "cre"),
-    email = "will.landau@gmail.com",
-    comment = c(ORCID = "0000-0003-1878-3253")
-  ),
-  person(
-    given = "Kevin",
-    family = "Kunzmann",
-    role = "aut",
-    email = "kevin.kunzmann@boehringer-ingelheim.com",
-    comment = c(ORCID = "0000-0002-1140-7143")
-  ),
-  person(
-    given = "Yoni",
-    family = "Sidi",
-    role = "aut",
-    email = "yoni.sidi@sagerx.com"
-  ),
-  person(
-    given = "Christian",
-    family = "Stock",
-    role = "aut",
-    email = "christian.stock@boehringer-ingelheim.com",
-    comment = c(ORCID = "0000-0002-3493-3234")
-  ),
-  person(
-    family = "Eli Lilly and Company",
-    role = c("cph", "fnd")
-  ),
-  person(
-    "Boehringer Ingelheim Pharma GmbH & Co. KG",
-    role = c("cph", "fnd")
-  ))
 Depends:
-  R (>= 4.0.0)
+    R (>= 4.0.0)
 Imports:
-  brms (>= 2.19.0),
-  coda,
-  dplyr,
-  emmeans (>= 1.8.7),
-  ggplot2,
-  ggridges,
-  MASS,
-  posterior,
-  purrr,
-  rlang,
-  stats,
-  tibble,
-  tidyr,
-  tidyselect,
-  trialr,
-  utils,
-  zoo
+    brms (>= 2.19.0),
+    coda,
+    dplyr,
+    emmeans (>= 1.8.7),
+    ggplot2,
+    ggridges,
+    MASS,
+    posterior,
+    purrr,
+    rlang,
+    stats,
+    tibble,
+    tidyr,
+    tidyselect,
+    trialr,
+    utils,
+    zoo
 Suggests:
-  BH,
-  knitr (>= 1.30),
-  markdown (>= 1.1),
-  Rcpp,
-  RcppEigen,
-  RcppParallel,
-  rmarkdown (>= 2.4),
-  rstan,
-  StanHeaders,
-  testthat (>= 3.0.0)
+    BH,
+    gt,
+    gtsummary,
```

Comment:
I do realize packages like `gt` and `gtsummary` are needed for the vignette, but it would be great to keep the original formatting and indentation of the DESCRIPTION file.

Reply:
Yes, no problem. I did not even notice the change. It might have been the effect of `usethis::use_package("...", "suggests")`.

Reply:
Done in commit 6efc7eeda36fd24259ff995dcfe682846f8c05eb.